src/main/kotlin/com/autonomousapps/tasks/ComputeUsagesTask.kt

```diff
@@ -130,7 +130,15 @@ private class GraphVisitor(
     var hasSecurityProvider = false
     var hasNativeLib = false
 
-    dependency.capabilities.values.forEach { capability ->
+    var capabilities = dependency.capabilities
+
+    dependencyCoordinates.gradleVariantIdentification.externalVariant?.let { externalVariant ->
+      // If this is just a shim to an external variant, defer to its capabilities instead
+      val externalIdentifier = externalVariant.capabilities.single()
```

Comment:
wrt to `single()`, do we know that there's only one capability here?

Reply:
We do yeah 👍